sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/AfterProcessingTime.java

```diff
@@ -60,17 +107,28 @@ public Instant getWatermarkThatGuaranteesFiring(BoundedWindow window) {
 
   @Override
   protected Trigger getContinuationTrigger(List<Trigger> continuationTriggers) {
-    return new AfterSynchronizedProcessingTime();
+    return AfterSynchronizedProcessingTime.ofFirstElement();
   }
 
   @Override
   public String toString() {
     StringBuilder builder = new StringBuilder("AfterProcessingTime.pastFirstElementInPane()");
-    for (SerializableFunction<Instant, Instant> delayFn : timestampMappers) {
-      builder
-          .append(".plusDelayOf(")
-          .append(delayFn)
-          .append(")");
+    for (TimestampTransform transform : getTimestampTransforms()) {
+      if (transform instanceof TimestampTransform.Delay) {
+        TimestampTransform.Delay delay = (TimestampTransform.Delay) transform;
+        builder
+            .append(".plusDelayOf(")
```

Comment:
Agreed. The fact that triggers try to `toString()` like their static factory methods is not something I care deeply about. But the logic I've added here really is just to keep parity (modulo the fact that the code wasn't really complete previously). OK to defer that? There will surely be another PR converting triggers to autovalue, which seems like a good time to do that kind of cleanup.